test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/DeepInsertSaveResultTests.cs

```diff
@@ -0,0 +1,316 @@
+﻿//---------------------------------------------------------------------

```

Comment:
Could you add tests that verify that `ReplaceUpdate` and `PostOnlySetProperties` are handled correctly?

Reply:
Could you also add a test that deep inserts and entity that has direct or indirect link to an entity that's neither new, modified nor deleted, i.e. an entity that had already been synced with the server before the inserted entity was attached to the tracker

Reply:
Actually we shouldn't be supporting `ReplaceUpdate`and `PostOnlySetProperties` in deep insert. So I have removed the codes